### ISSUE TITLE
Any instruction on shared-file-system setting?

### ISSUE THREAD

**Issue by RogerXT** [codalab/codalab-worksheets#436]
Will the worker still upload files to bundle service? Or the worker will just copy the files to bundle service?

**Comment by RogerXT** [comment]
When I use shared-file-system, I can see that there will be a new folder .codalab/partitions/default/

Then if I run a job, there's not output.

Without shared-file-system setting, the worker will download files to its scratch folder and execute the job, I just want to know with shared-file-system, what the worker will do to run the job.

Thanks a lot!

**Comment by percyliang** [comment]
If you use a shared file system, both the worker and bundle service will
read/write directly to `.codalab/partitions/default` without any
intermediate file transfers.

**Comment by RogerXT** [comment]
@percyliang Thanks a lot!
I figured it out by myself. Previously when I was using this setting, It won't work. 
I found that in codalab_manager, it will looking for self.config['workers'].get('shared_file_system', False).
I don't know what self.config['workers'] is, so I just hard-coded 
self._worker_model.shared_file_system = True
inside bundle_service. At least it worked.

One thing that not perfect is, on website, previously all bundles will have a metadata, data_size, which is computed after uploading. Now there's no uploading of a finished running bundle, so I didn't know how big is this running bundle. It won't affect the functionality, though.

I am not using Docker, I am not sure whether using Docker will compute data_size automatically. Maybe I can find a way to update data_size.
